rpc.js

```diff
@@ -1,7 +1,7 @@
 const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
can use normal import for fetch module if not using common js

Reply:
addressed